ansible_image_validation/validation-playbooks/blacklisted_drivers_validation.yaml

```diff
@@ -0,0 +1,66 @@
+###
+# This Playbook checks if the some specific drivers are blacklisted in the given image
+# Author: Anuj Maurya  
+###
+---
+
+- name: Print RHEL minor version
```

Comment:
ansible_distribution_version would provide both major and minor version as in "7.9". So, name can be 'RHEL Version'

Reply:
Add minor version comments specifically to help developers understand what we are looking for in this task.
We have another variable to check RHEL minor version in Ansible